app/assets/javascripts/controllers/reconfigure/reconfigure_form_controller.js

```diff
@@ -44,6 +46,20 @@ ManageIQ.angular.app.controller('reconfigureFormController', ['$http', '$scope',
       .catch(miqService.handleFailure);
   };
 
+  vm.setEnableAddDiskButton = function () {
+    var nrDisksAfterReconfigure=0;
+    angular.forEach(vm.reconfigureModel.vmdisks, function(disk){
+        switch (disk.add_remove) {
```

Comment:
Wrong indentation ;)

Please see the rest of the code climate warnings - feel free to ignore `Function updateDisksAddRemove has 36 lines of code (exceeds 25 allowed). Consider refactoring. ` but the rest is definitely relevant :).